src/views/modal/registration-js.php

```diff
@@ -10,20 +10,24 @@
  * @version TBD
  *
  */
+/** @var Tribe__Tickets__Attendee_Registration__View $view */
+$view = tribe( 'tickets.attendee_registration.view' );
+/** @var Tribe__Tickets__Editor__Template $template */
+$template = tribe( 'tickets.editor.template' );
 
 // There should be only one!
 $providers             = wp_list_pluck( $tickets, 'provider' );
 $providers_arr         = array_unique( wp_list_pluck( $providers, 'attendee_object' ) );
 $provider              = $providers[0];
-$provider_class        = tribe( 'tickets.attendee_registration.view' )->get_form_class( $providers_arr[0] );
+$provider_class        = $view->get_form_class( $providers_arr[0] );
 $has_tpp               = in_array( Tribe__Tickets__Commerce__PayPal__Main::ATTENDEE_OBJECT, $providers, true );
 $event_id              = get_the_ID();
 $meta                  = Tribe__Tickets_Plus__Main::instance()->meta();
 $non_meta_count        = 0;
 ?>
 <div class="tribe-tickets__item__attendee__fields">
 	<h2 class="tribe-common-h3 tribe-common-h4--min-medium tribe-common-h--alt tribe-tickets__item__attendee__fields__title"><?php esc_html_e( 'Attendee Details', 'event-tickets' ); ?></h2>
-	<?php tribe( 'tickets.editor.template' )->template(
+	<?php $template->template(
```

Comment:
`$this`?

Reply:
nope - this gets used in the modal and the modal doesn't have the template function carry through.